### ISSUE TITLE
how to use it in an irregular domain

### ISSUE THREAD

**Issue by WeCcRy** [benmoseley/FBPINNs#16]
I want to know that how to use it in a irregular domain and could you provide an example. By the way,could you explain more about the following code. I can't figure out that how it work and what kind of input should i provide(like use it to a complex boundary condition). Thanks.
` # boundary loss
    x_batch_boundary = jnp.array([0.]).reshape((1,1))     
    u_boundary = jnp.array([1.]).reshape((1,1))     
    ut_boundary = jnp.array([0.]).reshape((1,1))      
    required_ujs_boundary = (
        (0,()),
        (0,(0,)),
    )
`
